modules/apps/app-builder/app-builder-lang/src/main/resources/content/Language.properties

```diff
@@ -48,7 +48,7 @@ filter-entries=Filter Entries
 filter-entries-by-columns=Filter Entries by Columns
 filter-entries-by-setting-a-value-for-this-column=Filter entries by setting a value for this column.
 filter-entries-by-setting-filters-for-each-column=Filter entries by setting filters for each column.
-for-all-forms-of-this-object=For All Forms of This object
+for-all-forms-using-this-field=For all forms using This Field
```

Comment:
@jeyvison could you help me with this issue? When I run `gw buildLang` the word `This` is capitalized.

Reply:
Hey @interaminense , since it was SF that changed it's fine. Or you can try to ask on the sf channel too but i wouldn't worry about it ^^